neptune/client.py

```diff
@@ -667,9 +713,14 @@ def _upload_loop_chunk(self, fun, part, data, **kwargs):
                    },
                    **kwargs)
 
-    def _upload_raw_data(self, experiment, api_method, data, headers):
-        url = self.api_address + api_method.operation.path_name
-        url = url.replace("{experimentId}", experiment.internal_id)
+    def _upload_raw_data(self, api_method, data, headers, path_params, query_params):
+        url = self.api_address + api_method.operation.path_name + "?"
+
+        for key, val in path_params.iteritems():
+            url = url.replace("{" + key + "}", val)
+
+        for key, val in query_params.iteritems():
+            url = url + key + "=" + val + "&"
```

Comment:
this will produce a query ending with '&'

Reply:
Yeah, and it's totally fine.